RULES/core/util.go

```diff
@@ -54,12 +57,20 @@ func Fatal(format string, a ...interface{}) {
 	os.Exit(1)
 }
 
+func strHasSuffix(str, suffix string) bool {
```

Comment:
why?

Reply:
can't you use strings.HasSuffix in the FuncMap?

Reply:
duh, yes :)